src/store/notifications/notifications.action.js

```diff
@@ -0,0 +1,26 @@
+import { NOTIFICATION_ADDED } from './notifications.reducer';
+import { $injector } from '../../injection';
+import { EventLike } from '../../utils/storeUtils';
+
+const getStore = () => {
+	const { StoreService } = $injector.inject('StoreService');
+	return StoreService.getStore();
+};
+
+/**
+* @typedef {Object} Notification
+* @property {string} message the notification message
+* @property {enum} level the notification level
+* @property {Boolean} permanent whether thethe notification level
+ */
+
+/**
+ * sets the current notification
+ * @param {Notification} notification
+ */
+export const setNotification = (notification) => {
```

Comment:
Would suggest something like 'emitN...', 'publishN...',or 'fireN...' (because it's more like an event)'